recipes/bitshuffle/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "bitshuffle" %}
+{% set version = "0.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kiyo-masui/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c3f4461d8013e3d9db0d58defec77b143164652de505a1fba3df088eaa19be2f
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - cython
```

Comment:
If you're using `cython` you probably [need a compiler](http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=compiler#compilers):

Reply:
great! thanks @chrisburr ! I will try that!